src/submission/validation-clinical/utils.ts

```diff
@@ -288,31 +291,17 @@ const getSubmissionErrorInfoObject = (
 // ******* common resued functions *******
 export function treatmentTypeNotMatchTherapyType(
   treatmentType: string,
-  therapyType: ClinicalEntitySchemaNames,
+  therapyType: ClinicalTherapyType,
 ): boolean {
-  treatmentType = treatmentType.toString().toLowerCase();
-
-  if (therapyType === ClinicalEntitySchemaNames.CHEMOTHERAPY) {
-    return (
-      treatmentType !== 'chemotherapy' &&
-      treatmentType !== 'combined chemo+immunotherapy' &&
-      treatmentType !== 'combined chemo+radiation therapy' &&
-      treatmentType !== 'combined chemo-radiotherapy and surgery'
-    );
-  }
-
-  if (therapyType === ClinicalEntitySchemaNames.RADIATION) {
-    return (
-      treatmentType !== 'radiation therapy' &&
-      treatmentType !== 'combined chemo+radiation therapy' &&
-      treatmentType !== 'photodynamic therapy'
-    );
-  }
+  return !TreatmentTypeValuesMappedByTherapy[therapyType].find(
+    ttv => ttv.toString().toLowerCase() === treatmentType.toString().toLowerCase(),
```

Comment:
I'm not sure if we should lower case here, at this point it should've been normalized, did you encounter a case or think this is needed ?

Reply:
Yea ideally the record should match the codelist values by this point. I just did to `toLlowerCase()` to avoid potential bug. I'm not married to it I can remove it if you prefer.